RPIPs/RPIP-59.md

```diff
@@ -58,22 +56,25 @@ ETH from the deposit pool SHALL be matched with validator deposits from queues a
 - `deposit` SHALL assign deposits as described below 
 
 #### Assigning ETH from the Deposit Pool
-- As ETH enters the deposit pool, it SHALL be assigned to valdiators from the queue by sending 32 ETH to the associated megapool contract
+- ETH from the deposit pool SHALL be assigned to validators to validator at the front of the queue by sending 32 ETH to the associated megapool contract
+  - rETH mints SHALL assign `ETH_deposit // 32` validators
```

Comment:
Didn't recognize "//", looked it up and seems like it's a python thing (floor division)? Seems like solidity by default is floor division for "/"? Not sure if it's sufficient to just do "/" or if any other explicit description is needed

Reply:
I think usage of "/" in a written spec like here is likely to be understood as regular division and not floor division. We can probably even find examples of that usage of "/" in existing RPIPs. If "//" isn't understood commonly, we should say "floor division" explicitely.

Reply:
Is it being used as an escape character?

Reply:
I think we should just say "floor division" explicitly then

Reply:
> Is it being used as an escape character?

No, samsus is correct, it means floor division here.

Reply:
Going with 
> - rETH mints SHALL assign `floor(ETH_deposit / 32)` validators